packages/common/src/definitions/modifiers.ts

```diff
@@ -0,0 +1,126 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+
+import { objectFromKeys, swapKeyAndValue } from "../util/objectUtils";
+
+const modifiers = {
+  bounds: "excludeInterior",
```

Comment:
I would have mapped id to spoken form, as the former is the primary key for that entity. How come you went this way?

Reply:
This is the way we are used to seeing them in Talon. I'm also thinking that we're going to use the same source to actually generate the spoken forms as well as cheatsheet. In my mind this just feels "right".

Reply:
All true.